module-apps/application-phonebook/widgets/InputLinesWithLabelWidget.cpp

```diff
@@ -303,5 +309,14 @@ namespace gui
             return contact->note != inputText->getText();
         };
     }
+    void InputLinesWithLabelWidget::setRightFunctionKayLabel()
+    {
+        if (inputText != nullptr && inputText->isInputMode(InputMode::phone) && this->navBarSetRFKeyLabel &&
+            !inputText->isEmpty()) {
+            this->navBarSetRFKeyLabel(utils::translate(utils::translate("app_call_clear")), true);
+            return;
```

Comment:
It'd be better to use `else` statement for `navBarSetRFKeyLabel` instead of using `return`. I suppose that you don't need to use `this` after all ;)